src/scss/_functions.scss

```diff
@@ -1,18 +1,13 @@
-////
-/// @group o-fonts
-/// @link http://registry.origami.ft.com/components/o-fonts
-////
-
 /// Check if a font variant exists for a family
 ///
-/// @param {String} family - one of $o-fonts-families
-/// @param {String} weight
-/// @param {String} style
-///
-/// @access private
-///
+/// @access public
+/// @param {String} $family - one of $o-fonts-families
+/// @param {String} $weight [regular] - The font weight.
+/// @param {String} $style [normal] - The font style.
 /// @return {Bool}
-@function _oFontsVariantExists($family, $weight, $style) {
+@function oFontsVariantExists($family, $weight: 'regular', $style: 'normal') {
+	$weight: if($weight, $weight, 'regular');
```

Comment:
Do we need this `if` statement if the default `$weight` is 'regular'?
Same with the `$style` on the line below?

Reply:
I wanted to allow passing in a falsy value like null/false as in "no weight". I was thinking about this case https://github.com/Financial-Times/o-typography/pull/163/files#diff-b2aec7b3cdbb70a5b21c2660f07c4c4cR19

Reply:
What for though? What is 'no weight'?

Reply:
I mean none specified, so the default 'regular' should be used.